components/homepage/sections/KeyFeatures.tsx

```diff
@@ -0,0 +1,154 @@
+import { Aurora } from "../Aurora";
+import { Box, Image, SimpleGrid } from "@chakra-ui/react";
+import { HomepageSection } from "components/product-pages/homepage/HomepageSection";
+import { ReactNode } from "react";
+import { Heading, Link, Text } from "tw-components";
+
+interface FeatureCardProps {
+  title: string;
+  description: ReactNode;
+  img: string;
+}
+
+const features: FeatureCardProps[] = [
+  {
+    title: "Build.",
+    description: (
+      <>
+        Build your own contracts with{" "}
+        <Link
```

Comment:
would be great if these would be `<TrackedLink>` instead of `<Link>`

Reply:
Fixed in https://github.com/thirdweb-dev/dashboard/pull/770/commits/9716a32ccb6489b24fc6f8462beaee083d09f887